db/migrate/20221125173217_remove_enable_whispers_site_setting.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class RemoveEnableWhispersSiteSetting < ActiveRecord::Migration[7.0]
+  def up
+    execute <<~SQL
```

Comment:
might be interesting to comment the meaning of 3 here, it's more or less clear today, but might lose meaning in years